projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/api/meta_job.py

```diff
@@ -16,6 +18,7 @@ class SingleJob:
     job_name: str
     team_name: str = None
     fail_meta_job_on_error: bool = True
+    arguments: IJobArguments = None
```

Comment:
We use primitive types here only. IJobArguments is interface for arguments to be read from Job Input. 

The arguments itself are of type "dict" , we even validate there are of type dict. While IJobArguments is another interface that is not compatible with dict.

Reply:
Makes sense. Done.